packages/cashier-v2/src/containers/Cashier/Cashier.tsx

```diff
@@ -35,18 +30,16 @@ const Cashier: React.FC<TRouteTypes.TRouteComponent> = ({ routes }) => {
                     </VerticalTab>
                 }
             >
-                <CashierLayout sideNotes={sideNotes}>
-                    <Switch>
-                        {routes?.map(route => {
-                            const { path, title } = route;
-                            return (
-                                <Route exact key={path} path={path}>
-                                    <route.component path={path} setSideNotes={setSideNotes} title={title} />
-                                </Route>
-                            );
-                        })}
-                    </Switch>
-                </CashierLayout>
```

Comment:
Here we are removing `CashierLayout` and rename it to generic `PageContainer` that we will use directly in our `flows` (pages, e.g Deposit, Withdrawal, etc.).